tests/test_lane.py

```diff
@@ -3099,7 +3111,7 @@ def setup(self):
         random.seed(42)
 
     def test_groups(self):
-        """A comprehensive test of WorkList.groups()"""
+        # A comprehensive test of WorkList.groups()
```

Comment:
I changed these to comments when trying to debug the mysterious test failures. These are the tests that were failing.